src/framework/piral-base/src/strategies.test.ts

```diff
@@ -196,6 +196,7 @@ describe('Piral-Base strategies module', () => {
 
     // Act
     await expect(blazingStrategy(invalidLoadPiletOptions, callbackMock)).rejects.toThrowError();
+    // expect(failed).toBeTruthy();
```

Comment:
```suggestion
```

I think this is not supposed to be there (I cannot see any failed variable, at least)..